src/main/java/se/sundsvall/contract/api/model/Contract.java

```diff
@@ -30,7 +30,7 @@
 public abstract class Contract {
 
 	@Schema(description = "Contract id", example = "2024-12345", accessMode = Schema.AccessMode.READ_ONLY)
-	private String id;
+	private String contractId;
```

Comment:
Isn't `id` sufficient? The prefix "contract" seems reduntant in a class named "Contract"

Reply:
Will keep it as it is for now if that's ok.
Main reason being that there are a lot of `id`:s in circulation in the API and this helps keep them separated. Otherwise I agree :)